molecule/default/templates/awx_cr_molecule.yml.j2

```diff
@@ -27,3 +27,4 @@ spec:
       memory: 32M
   postgres_resource_requirements: {}
   postgres_init_container_resource_requirements: {}
+  redis_resource_requirements: {}
```

Comment:
I set our redis resource requirements to an empty dictionary here so that the pods can be scheduled on github workflows as there are few resources available and the pods were not getting scheduled.